### DISCUSSION TITLE
Welcome to Python dactyl-keyboard Discussions!

### DISCUSSION THREAD

**Discussion by joshreve** [joshreve/dactyl-keyboard#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ja2ui0** [comment]
I read this intro after I posted my first two comments, sorry! As I have another conversation to start, here's a quick intro.

I'm a lifetime nerd and professional open source / linux guy. For money, I am essentially a technology partner and support liaison for companies utilizing kubernetes based solutions in their enterprise software offerings. For fun, I'm a dad and husband first and maker second. I feel that the maker part is closely coupled with being a dad.

In the mechanical keyboard space, five years or so ago I saw my at the time manager display a handwired keyboard he made from wood. At the time, I was not aware that humans did that kind of thing. This led me to Jack Humbert's OLKB projects, and I built a Preonic from a kit on (mass)drop. It was my daily driver for a few months, and during that time I learned a few things about my hands and small keyboards. Also that I like ortholinear layouts. So I decided that someday when I had money to blow I'd get an ergodox and that would be my end game keyboard.

Recently a friend at work acquired a Dactyl Manuform from mechmarket, and immediately I knew that I was not getting an ergodox, as I recently became a 3d printer enthusiast and this model ticks boxes I never knew I had. So here I am, participating in this community because I am not a passive consumer.
